version_information_endpoint.asciidoc

```diff
@@ -215,4 +215,4 @@ In this case the `credentials` module is not defined twice as this module is the
 [source,json]
 ----
 include::examples/version_details_example2.json[]
-----
\ No newline at end of file
+----
```

Comment:
What changed here? Seems like the same line, watch out with adding/removing white spaces or other non printed characters.